.github/workflows/test.yml

```diff
@@ -31,6 +31,13 @@ jobs:
         source /opt/ros/$ROS_DISTRO/setup.bash
         colcon build --symlink-install
         source install/local_setup.bash
+    - name: Test Package
+      run: |
+        source /opt/ros/$ROS_DISTRO/setup.bash
+        source install/local_setup.bash
+        # skip testing submodules
+        colcon test --packages-skip bdai_ros2_wrappers --event-handlers console_cohesion+
```

Comment:
- Prefer `console_direct` vs `console_cohesion`
- Consider adding` --python-testing pytest --pytest-with-coverage` and add integration for Codecov
- Add a second line` colcon test-result --all --verbose`